docs/manual/developer/03_creating_content.md

```diff
@@ -355,3 +355,372 @@ A profile should define these attributes:
     or
 
 -   rule refinements, e.g. `accounts_tmout.severity=high`.
+
+
+## Controls
+
+The controls adds another layer on top of profiles. Controls files store the
+metadata for security controls and, more importantly, concentrate the mapping
+from requirement to rule at a single place.
+
+We will explain the format using the expected workflow and we will show examples
+of the format.
+
+### Storing controls data
+
+When we develop a new SCAP profile, usually we first get a PDF document with a
+policy that consists of controls (requirements). For example NIST 800-53
+controls, ANSSI requirements, CIS Benchmark, etc.
+
+To add the policy to our project repository, we will create a YAML
+file that represents this policy. There is a special directory, called
+`controls`, in the repository root to store these files. These files serve as a
+database of controls (requirements). They are independent from profiles and
+products. We can extract the interesting data from the PDF and save them in a
+YAML file. We can also create this file manually, or it can be converted from
+some other format (XML, OpenControl) if available and then submitted.
+
+There is a YAML file that represents the whole policy. Initially, the
+YAML file can look like this:
+
+```
+$ cat controls/abcd.yml
+
+id: abcd
+title: ABCD Benchmark for securing Linux systems
+version: 1.2.3
+source: https://www.abcd.com/linux.pdf
+controls:
+  - id: R1
+    title: User session timeout
+    description: |-
+      Remote user sessions must be closed after a certain
+      period of inactivity.
+  - id: R2
+    title: Minimization of configuration
+    description: |-
+      The features configured at the level of launched services
+      should be limited to the strict minimum.
+  - id: R3
+    title: Enabling SELinux targeted Policy
+    description: |-
+      It is recommended to enable SELinux in enforcing mode
+      and to use the targeted policy.
+```
+
+In the real world, controls (requirements) can be nested. For example, PCI-DSS
+has tree structure, within requirement 2.3 we can find 2.3.a, 2.3.b, etc.
+Therefore, each item in `controls` list can contain a `controls` list.
+
+Once we have the file, we can read through the policy requirements and assess
+each requirement. For the controls we will have to identify whether it can be
+automated by SCAP. If yes, we should look if we already have existing XCCDF
+rules in our project. 
+
+For example, let’s say that we identified that:
+
+* R1 can be automatically scanned by SCAP and we already have 3 existing rules
+in our repository. However, we want one of them to be selected only on RHEL 9,
+but the rule is applicable to all platforms.
+* R2 is up to manual checking, but we have systemd_target_multi_user which is
+related to this control.
+* R3 can be automatically scanned by SCAP but unfortunately we don’t have any
+rules and checks implemented yet.
```

Comment:
Isn't "and checks" redundant as we'd have no way to have a check independent of a rule but still satisfying the control? That is, we could put a check under `shared/...` but we'd have no way of invoking it for the specified control without a rule entry?

Reply:
But if you want to keep it: nit: `or checks` rather than `and checks` makes more sense to me.